win32ss/gdi/dib/dib16bpp.c

```diff
@@ -6,13 +6,17 @@
  * PROGRAMMERS:     Jason Filby
  *                  Thomas Bluemel
  *                  Gregor Anich
+ *                  Doug Lyons
  */
 
 #include <win32k.h>
 
 #define NDEBUG
 #include <debug.h>
 
+#define DEC_OR_INC(var, decTrue, amount) \
```

Comment:
Instead of this macro, I suggest using a delta variable and setting it to +1 / -1 at the function start.

Reply:
In spite of the reference to "Dec" and "Inc" in the name, the amount changes over a wide range throughout the code. So the reference to "+1/-1" would not work in many cases. Maybe the name of the macro should be changed to "ADJUST"? Thanks for your comments.

Reply:
It was mostly a matter of performance. This will check a variable and then do add or sub. If you saved the value, if would only be an add. This should work with any value, like it's done with xinc and ySrcDelta. Anyway, I don't consider it a must and I don't want to hold this back unnecessarily. It can still be optimized later.